great-idea-website/css/index.css

```
@@ -63,4 +63,260 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+
+/* Your code starts here! */
+*{
+	box-sizing: border-box;
+}
+
+body{
+	width:800px;
+	margin: 0 auto;
+}
+.container{
+	/* border: 1px solid red; */
+	width: 800px;
+	margin:0 auto;
+	}
+
+.Header{
+	/* border: 1px solid blue; */
+	width: 100%;
+	display: inline-block;
+	position: relative;
+}
+
+ div .Navshell {
+	/* border: 1px solid blue; */
+	width: 100%;
+	display: inline-block;
+	margin-top: 15px;
+	line-height: 35px; 
+}
+
+nav{
+	/* border: 1px solid purple; */
+	/* background: lightgray; */
+	position: fixed;
+	display: inline-block;
+	z-index: 1;
+	}
+
+div nav a{
+	/* border: 1px solid green; */
+	margin: 15px 15px 40px 15px;
+	display: inline-block;
+	text-decoration: none;
+	color: grey;
+	font-size: 12px;
+	width: 60px;
+	
+}
+
+
+.Header .imglogo{
+	/* border: 1px solid purple; */
+	display: inline-block;
+	margin-right: 0px;
+	/* margin-left: 40px; */
+	padding-left: 40px;
+	padding-top: 30px;
+	position: absolute;
+	/* left: 900px;
+	top: 0; */
+
+}
+
+ .logo{
+	/* border: 1px solid orange; */
+	display: inline-block; 
+	margin-right: 0;
+	margin-left: 310px;
+	padding-top: 30px;
+	left: 625px;
+	top: 0;
+	position: absolute;
+	background-image: url("img/logo.png");
+ }
+
+.section1{
+	/* border: 1px solid purple; */
+	margin-top: 60px;
+
+	display: inline-block;
+	position: relative;
+	padding-top: 0;
+	border-bottom: 3px solid black;
+	width: 100%;
+	padding-bottom: 25px;
+}
+
+button{
+	text-align: center;
+	display: block;
+	width: 150px;
+	height: 30px;
+	margin-left: 55px;
+	background: white;
+	margin-top: 30px;
+}
+
```

To get your buttons to change style when you hover your mouse over them you could use:
```suggestion
button:hover {
    background: black;
    color: white;
}
```
when you see a : like that, it indicates a pseudo-class. Other common examples are :first-child and :last-child while will apply specific styles to only the first or last child elements.